client/src/components/Navbar/useStyles.ts

```diff
@@ -20,6 +20,19 @@ export const useStyles = makeStyles((theme: Theme) => ({
       color: theme.palette.primary.main,
     },
   },
+  unreadNotification: {
+    backgroundColor: '#FFFFFF',
+    fontWeight: 700,
+    textDecoration: 'none',
+  },
+  readNotification: {
+    backgroundColor: theme.palette.grey[300],
+    fontWeight: 700,
+    textDecoration: 'none',
+  },
```

Comment:
Can I have more clarification here? The docs are not clear to me.  Am I supposed to write a class within the useStyles component, and also pass props to that useStyles component, which are passed down from the notification elements as well? I also can't use
const classes = useStyles({read: notification.read}) because notification is undefined before I map the elements. 
I think my solution is easier to understand and is only one line of code in the Notification component but if its not how your supposed to conditionally render classes in MUI then I can adopt what your saying.

Reply:
@rj13371 "Can I have more clarification here? The docs are not clear to me. Am I supposed to write a class within the useStyles component, and also pass props to that useStyles component, which are passed down from the notification elements as well?" - Yes, we can use declare a CSS class in the useStyles file and conditionally apply styles by passing a prop from the Notifications component.

"const classes = useStyles({read: notification.read}) because notification is undefined before I map the elements." - We can perform a validity check and pass down the props only if a notification exists.

Reply:
You could also consider componentizing UI elements in these sort of cases 👍🏼

Reply:
I componentized read and unread notifications into their own components and just passed down the props from the Notification parent component.